enterprise/models.py

```diff
@@ -1477,6 +1508,7 @@ class EnterpriseCustomerReportingConfiguration(TimeStampedModel):
 
     class Meta:
         app_label = 'enterprise'
+        unique_together = (('enterprise_customer', 'data_type'),)
```

Comment:
@mattdrayer One more question: do you want this to be unique together with delivery method and report type as well? This'd allow reporting configs for a single customer for all combinations of data, report, and delivery types. E.g. Progress/Catalog CSV through SFTP/Email, which are 4 combinations, that are all simultaneously possible and will not conflict become of different filename schemes on the delivery end.

Reply:
That sounds amazingly awesome to me!